cirq/protocols/json_serialization.py

```diff
@@ -702,3 +703,41 @@ def obj_hook(x):
             return json.load(file, object_hook=obj_hook)
 
     return json.load(cast(IO, file_or_fn), object_hook=obj_hook)
+
+
+def to_gzip(
```

Comment:
This is a fairly generic name. Should it be `to_json_gzip`? `to_json_gz`? `to_json(..., compression='gzip')`? 

IMO the fact that it's a json document is more relevant than the fact that it's gzipped. You could have `to_pickle` or `to_proto`, each of which could also be gzipped (all hypothetically of course)

Reply:
Switched to `*_json_gzip`.